Jenkinsfile

```diff
@@ -1,21 +1,32 @@
+#!groovy
+
+@Library('github.com/mozmar/jenkins-pipeline@master')
```

Comment:
`github.com/mozmar` -> `github.com/mozmeao`

Reply:
Good catch, I'll change that, thanks @metadave!